### DISCUSSION TITLE
【求助】这个到底是驱动问题还是显卡硬件问题啊！！

### DISCUSSION THREAD

**Discussion by kether123** [share121/inter-knot#1755] [其他]
这几天发现电脑出现了类似图中的花屏现象，尤其屏幕上出现蓝色时，花屏会更严重甚至黑屏，在某QQ群里都说是显卡问题，我虽然觉得也是这样，但却不清楚是驱动问题还是硬件问题，如果是驱动问题好歹还可以修，如果是硬件问题那只能换了啊...哪位大佬可以说下这是什么问题？
![picimage](https://github.com/user-attachments/assets/3b9d6074-b361-4040-a9d2-2dd31d10b4be)

**Comment by capacitor1** [comment]
🤔猜测大概率是硬件损坏。驱动出问题好像不是这种故障

看这种情况估计不是屏幕磕烂了就是显存问题？

**Comment by SnowCandle12138** [comment]
选择你的排名助手（
![Uploading 495f12a4d0edc698c0e5a6437efdfa6b.png…]()

**Comment by SnowCandle12138** [comment]
可恶、绳网还在维护吗、我发的评论照片看不了、希望可以完善一下

**Comment by share121** [comment]
你要等图片上传完成后才能提交，显示为 `Uploading ` 说明没有上传完成

**Comment by cpu644** [comment]
这样花了就是显存寄了或者屏幕寄了 试试从nv/amd官网自己找最新驱动打上 还不行换个显示器（借一个）再不行就只能找维修师傅了 显存一般换不会太贵 不过也不排除别的可能性

**Comment by lamtt26** [comment]
不道啊，不过你也有mumu,欸嘿

**Comment by Rain-YYC** [comment]
有可能是屏幕问题，显卡的话可能性不大（猜的，试试装最新的显卡驱动看看问题好没好
